### PR TITLE
treewide: Remove IFDs

### PR DESCRIPTION
**Pull request by lorenzleutgeb**
Relates to:
 - https://github.com/ngi-nix/ngipkgs/issues/24
 - https://github.com/ngi-nix/ngipkgs/issues/39
 - https://github.com/ngi-nix/ngipkgs/issues/40

### CODE REVIEW

File: pkgs/kikit/solidpython/default.nix [hunk 1]
```diff
@@ -62,13 +41,18 @@ in
       hash = "sha256-3fJta2a5c8hV9FPwKn5pj01aBtsCGSRCz3vvxR/5n0Q=";
     };
 
+    patches = [
+      # Unpin PrettyTable, see <https://github.com/SolidCode/SolidPython/issues/207>.
+      ./prettytable.patch
+    ];
```
Comment (jfly) [inline pkgs/kikit/solidpython/default.nix:47]: We learned recently that nixpkgs actually has a more idiomatic way of relaxing dependencies. See https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook, and us using it here: https://github.com/NixOS/nixpkgs/pull/249464/commits/38a7f42312ad59e7604dc1ed01ebc1acaf59849c.

**Review by jfly — APPROVED** [review]
The kikit stuff LGTU (@ngi-nix/fungus)! Thanks for dealing with it. We actually had to remove IFD when we submitted this stuff to try to upstream it to nixpkgs in https://github.com/NixOS/nixpkgs/pull/249464/, sorry for the duplicated work!

File: pkgs/kikit/solidpython/default.nix (hunk 1 above)
Comment (lorenzleutgeb) [reply pkgs/kikit/solidpython/default.nix:47]: Nice, thanks! This might come in handy also for packaging pretalx. I'll update this PR ASAP.

Comment (lorenzleutgeb) [reply pkgs/kikit/solidpython/default.nix]: Fixed in 35b6a22.

**Review by cleeyv — COMMENTED** [review]
This generally looks good to me! The only thing I see missing is to replace the use of liberaforms as an example in the README for adding and deploying a service: https://github.com/ngi-nix/ngipkgs/blob/main/README.md?plain=1#L43-L49

I think pretalx would be an excellent alternative to liberaforms in this section. ~~It doesn't need to be exactly the same things covered, you can just copy what you've already put in your README for pretalx, so it could be changed to "Add and test a service" and we can include more about deployment later.~~

I'm working now on making this edit to the README using the pretalx example, hopefully a commit coming soon.

**Review by cleeyv — APPROVED** [review]

**Comment by mightyiam** [comment]
Nooice

**Pull request merged**